JavaScript/JavaScript.sublime-syntax

```diff
@@ -2053,7 +2053,7 @@ contexts:
       scope: support.class.builtin.js
       pop: true
     - match: (?:Eval|Range|Reference|Syntax|Type|URI)?Error{{identifier_break}}
-      scope: support.class.error.js
+      scope: support.class.builtin.js
```

Comment:
Would it make sense to merge the pattern with the former one then?

Reply:
It would be compiled the same either way, so it shouldn't really matter.